presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -109,6 +109,24 @@ Name                                   Description
 ``KerberosKeytabPath``                 Kerberos keytab file.
 ``KerberosCredentialCachePath``        Kerberos credential cache.
 ``extraCredentials``                   Extra credentials for connecting to external services. The
-                                       extraCredentials is a list of key-value pairs. Example:
+                                       ``extraCredentials`` is a list of key-value pairs. Example:
                                        ``foo:bar;abc:xyz`` will create credentials ``abc=xyz`` and ``foo=bar``
+``roles``                              Authorization roles that will be set for catalogs. The
+                                       ``roles`` is a list of key-value pairs, where key represents a reference to 
+                                       Presto catalog and value represents role to be set. Example:
+                                       ``catalog1:none;catalog2:all;catalog3:role`` will set:
+
+                                       - ``none`` role in ``catalog1``
+                                       - ``all`` role in ``catalog2``
+                                       - ``role`` role in ``catalog1``
+
+``sessionProperties``                  Session properties that will be set globally and for catalogs. The
```

Comment:
A list of key-value pairs to set session properties globally and specifically for catalogs. The format for global properties is ``propertyname:value``. Catalog-specific properties use the catalog name as a prefix ``catalog.propertyname:value``. Multiple properties are separated by ``;``.

Reply:
Similarly, I like the original better.
```
Session properties to set for the system and for catalogs, specified as a list of key-value pairs.
For example, ``abc:xyz;example.foo:bar`` will set the system property ``abc`` to the value ``xyz``
and the ``foo`` property for catalog ``example`` to the value ``bar``.

Reply:
again .. no future tense. Otherwise this is also fine by me.